openssl/aes.go

```diff
@@ -157,17 +135,7 @@ func (x *aesCBC) CryptBlocks(dst, src []byte) {
 		panic("crypto/cipher: output smaller than input")
 	}
 	if len(src) > 0 {
-		outlen := C.int(0)
-		// Workaround - padding detection is broken but we don't need it
-		// since we check for full blocks
-		if C._goboringcrypto_EVP_CIPHER_CTX_set_padding(x.ctx, 0) != 1 {
```

Comment:
@ueno @dbenoit17 I haven't found the need to implement this workaround. Could you give more context or share a failing test case?

Reply:
@derekparker could we move this `EVP_CIPHER_CTX_set_padding` to `NewCBCDecrypter` and `NewCBCEncrypter` so we only call it once per `aesCBC` instance instead of once per `CryptBlock` call?

Reply:
I think that should be fine.

Reply:
Done